src/mu-arr-int.c

```diff
@@ -0,0 +1,69 @@
+/* mu-str.c
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+#include <stdlib.h>
+
+#include "mu-arr-int.h"
+
+static MuArrInt * 
+mu_arrint_init ()
```

Comment:
It shouldn't be a static function or it won't be visible outside the file. We should probably follow the same naming style, so something like:

`mu_arr_int_new ()`

Reply:
Ah, sorry, I see what you were going for now. Should we keep the `_new/_add/_append` split?